### PR TITLE
Feature/menu link content entity type sql error

### PR DESCRIPTION
**Pull request by a-kom**

### CODE REVIEW

File: config/schema/menu_item_extras.utility.schema.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+menu_item_extras.utility:
```
Comment (iVegas) [inline config/schema/menu_item_extras.utility.schema.yml]: Move schema into menu_item_extras.schema.yml file

File: menu_item_extras.install [hunk 2]
```diff
@@ -38,19 +40,33 @@ function menu_item_extras_uninstall() {
...
   $mlc_helper->doEntityUpdate();
 }
 
+/**
+ * Implements hook_update_N().
+ *
+ * Trigger entity update for changes bundle in entity scheme.
+ */
+function menu_item_extras_update_8022() {
+  \Drupal::service('menu_item_extras.menu_link_content_helper')->updateMenuLinkContentBundle();
```
Comment (iVegas) [inline menu_item_extras.install]: If it is update path, initialize `entity_type_build_status` config

Comment (a-kom) [inline menu_item_extras.install]: By default it is TRUE, so we didn't need this.

Comment (iVegas) [inline menu_item_extras.install]: 76 Line: Looks good.

**Pull request merged**